pom.xml

```diff
@@ -335,6 +336,11 @@
           <version>2.17</version>
           <dependencies>
             <dependency>
+              <groupId>org.apache.apex</groupId>
+              <artifactId>apex-codestyle-config</artifactId>
+              <version>${project.version}</version>
```

Comment:
Yes, I think `apex-codestyle-config` should be a separate project (source code may still be part of Apex repository) with it's own version and release cycle. It then can be released to maven prior to making changes to Apex pom.